specification/monitor/resource-manager/readme.python.md

```diff
@@ -129,6 +150,11 @@ Please also specify `--python-sdks-folder=<path to the root directory of your az
 ``` yaml $(tag) == 'package-2019-10-17-preview-only'
 namespace: $(python-base-namespace).v2019_10_17
 output-folder: $(python-sdks-folder)/$(python-base-folder)/v2019_10_17
+directive:
```

Comment:
Not sure why remove `ResourceGroupNameParameter` here.

Reply:
moved it from specification/monitor/resource-manager/readme.md
it probably messed up the automatic sdk generation for the 2019 version. 

![image](https://user-images.githubusercontent.com/47111029/123535871-9220f400-d72f-11eb-9ba5-490ba46cd8e3.png)